### ISSUE TITLE
Settings windows take up to much room

### ISSUE THREAD

**Issue by mingomongo** [Cockatrice/Cockatrice#3951]
```
Client Version: 2.7.4 (2020-03-20)
Client Operating System: Windows 7 SP 1 (6.1)
Build Architecture: 64-bit
Qt Version: 5.12.6
System Locale: en_GB
Install Mode: Standard
```
_______________________________________________________________________________________

Hey, it's me again :) :100:
This time with something that's more of a suggestion, maybe this should be in feature requests, I dunno. Fact of the matter is, the setting window (on my Windows 7) is too damn big. As you can see it takes up almost half the screen for no real reason. The equal distance content spacing also amplifies this making it about 2 inches between menu items on the Sound menu. This isn't an issue on mac (well, there was one issue I posted about) but it looks relatively normal and compact there. It's still could be shorter, but it's alright. This one on windows is just too wide and too long.

In short, I think it's best to make 'Settings' take less unnecessary space by lowering minimum size(?) and switching to fixed distance item stacking because equal distance items looks dumb and serves no purpose.

![prettybig](https://user-images.githubusercontent.com/33768220/78839534-95c44000-79f0-11ea-80b0-cf0edc2cc05d.png)
![prettybigsound](https://user-images.githubusercontent.com/33768220/78839535-96f56d00-79f0-11ea-8be6-dc76eb68d82e.png)

**Comment by ctrlaltca** [comment]
Probably caused by https://github.com/Cockatrice/Cockatrice/pull/2513
Do we really set the height to monitor size - 100px? Sounds way too much :)
We should just define a fixed minimum, eg. 500px.
https://github.com/Cockatrice/Cockatrice/blob/master/cockatrice/src/dlg_settings.cpp#L1160

**Comment by tooomm** [comment]
On my small laptop screen the issue isn't present.
But yes... on a big screen it's stretched for no reason and eats up a lot of space! 

Win 10 (64bit) running 2.7.5-beta (2020-04-08)

**Comment by ZeldaZach** [comment]
This can probably be solved with an ifdef op for different OSes to solve it

**Comment by tooomm** [comment]
Are you sure the operting system is the issue here? @ZeldaZach 
Instead, it looks like the fixed minimum is weirdly calculated as mentioend by @ctrlaltca. That's probably why on small screens the issue is not present.

The reference in code should be this as it moved a bit in between:
https://github.com/Cockatrice/Cockatrice/blob/master/cockatrice/src/dlg_settings.cpp#L1194
> `this->setMinimumSize(rec.width() / 2, rec.height() - 100);`
